### PR TITLE
Add diagnostics using arviz to tutorials

### PR DESCRIPTION
**Pull request by sbidari**
Two changes made:
 - added mcmc diagnostics to the two tutorials
 - changed `num_chains=2` from deafult value of 1. This was required to suppress a warning about convergence diagnostics

### CODE REVIEW

File: model/docs/example_with_datasets.qmd [hunk 1]
```diff
@@ -238,7 +238,7 @@ hosp_model.run(
     num_warmup=2000,
     data_observed_hosp_admissions=dat["daily_hosp_admits"].to_numpy(),
     rng_key=jax.random.PRNGKey(54),
-    mcmc_args=dict(progress_bar=False),
+    mcmc_args=dict(progress_bar=False,num_chains=2),
```
Comment (dylanhmorris) [inline model/docs/example_with_datasets.qmd:241]: space 

```suggestion
    mcmc_args=dict(progress_bar=False, num_chains=2),
```

File: model/docs/example_with_datasets.qmd [hunk 2]
```diff
@@ -283,7 +283,7 @@ hosp_model.run(
     num_warmup=2000,
     data_observed_hosp_admissions=dat_w_padding,
     rng_key=jax.random.PRNGKey(54),
-    mcmc_args=dict(progress_bar=False),
+    mcmc_args=dict(progress_bar=False,num_chains=2),
```
Comment (dylanhmorris) [inline model/docs/example_with_datasets.qmd:286]:
```suggestion
    mcmc_args=dict(progress_bar=False, num_chains=2),
```

File: model/docs/getting_started.qmd [hunk 3]
```diff
@@ -192,7 +192,7 @@ model1.run(
     num_samples=1000,
     data_observed_infections=sim_data.observed_infections,
     rng_key=jax.random.PRNGKey(54),
-    mcmc_args=dict(progress_bar=False),
+    mcmc_args=dict(progress_bar=False,num_chains=2),
```
Comment (dylanhmorris) [inline model/docs/getting_started.qmd:195]:
```suggestion
    mcmc_args=dict(progress_bar=False, num_chains=2),
```

**Review by dylanhmorris — COMMENTED** [review]
A few minor code style things, otherwise LGTM

**Review by damonbayer — APPROVED** [review]
Looks good to me!

**Pull request merged**